docs/source/development/for_proj_contributors.rst

```diff
@@ -0,0 +1,83 @@
+.. _for_proj_contributors:
+
+================================================================================
+Development rules and tools for PROJ code contributors
+================================================================================
+
+This is a guide for PROJ, casual or regular, code contributors.
+
+Coding rules
+###############################################################################
+
+To be formalized. Current rule is do like existing surrounding code...
+
+Tools
+###############################################################################
+
+cppcheck static analyzer
+--------------------------------------------------------------------------------
+
+You can run locally scripts/cppcheck.sh that is a wrapper script around the
+cppcheck utility. It is known to work with cppcheck 1.61 of Ubuntu Trusty 14.0,
+since this is what is currently used on Travis-CI
+(travis/linux_gcc/before_install.sh).
```

Comment:
In other parts of the docs I've been trying to format paths in the repo as code by wrapping the text in \`\` . It would be good to do that here to so that it reads "(``travis/linux_gcc/before_install.sh``)"

Reply:
Fixed